x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/scalar/ScalarFunctionWritables.java

```diff
@@ -92,6 +93,7 @@ public static List<NamedWriteableRegistry.Entry> getNamedWriteables() {
         entries.add(Tau.ENTRY);
         entries.add(ToLower.ENTRY);
         entries.add(ToUpper.ENTRY);
+        entries.add(Hash.ENTRY);
```

Comment:
These too.

The alphabetical order helps make it slightly less likely we'll merge conflict with anyone else writing one of these at the same time. And a little easier to read.